index.html

```diff
@@ -1496,6 +1496,149 @@ <h3>Types</h3>
+are provided for these purposes.
+        </p>
+
+        <dl>
+          <dt><dfn class="lint-ignore">name</dfn></dt>
+          <dd>
+An OPTIONAL property that expresses the name of the <a>credential</a>. If
+present, the value of the <code>name</code> <a>property</a> MUST be a string or
+an object. If the value is an object, the object MUST contain a `value` property
+whose value is a string, and SHOULD contain a `lang` property whose value is a
+`langtag` string defined in [[BCP47]], and MAY contain a `dir` property whose
+value is a text direction string defined by the `dir` attribute in [[JSON-LD]].
+Ideally, the name of a <a>credential</a> is concise, human-readable, and could
+be used to enable an individual to quickly differentiate the <a>credential</a>
+from other <a>credentials</a> that they might hold.
+          </dd>
+          <dt><dfn class="lint-ignore">description</dfn></dt>
+          <dd>
+An OPTIONAL property that conveys specific details about a <a>credential</a>. If
+present, the value of the <code>description</code> <a>property</a> MUST be a
+string or an object. If the value is an object, the object MUST contain a
+`value` property whose value is a string, and SHOULD contain a `lang` property
+whose value is a `langtag` string defined in [[BCP47]], and MAY contain a `dir`
+property whose value is a text direction string defined by the `dir` attribute
+in [[JSON-LD]]. Ideally, the description of a <a>credential</a> is no longer than
+a few sentences in length and conveys enough information about the
+<a>credential</a> to remind an individual as to its contents without having to
+look through the entirety of the <a>claims</a>.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight vc"
+          title="Usage of the name and description property"
+          data-vc-vm="https://university.example/issuers/565049#key-1">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
+  "issuer": {
+    "id": "https://university.example/issuers/565049",
+    "name": "Example University",
+    "description": "A public university focusing on teaching examples."
+  },
+  "validFrom": "2015-05-10T12:30:00Z",
+  "name": "Example University Degree",
+  "description": "2015 Bachelor of Science and Arts Degree",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "subtype": "Bachelor of Science and Arts"
+    }
+  }
+}
+        </pre>
+
+        <p>
+Names and descriptions also support expressing content in different languages.
+To express a string with language and text direction information, one can use
+an object that contains the `value`, `lang`, and `dir` properties to express
+the text value, language code, and text direction, respectively.
```

Comment:
```suggestion
the text value, language, and text direction,
```

or perhaps:

```suggestion
the text value, language tag, and text direction,
```

Reply:
@aphillips -- Note that you can make it easier for PR authors to accept any suggestion from you by making each such suggestion in its own comment. GitHub provides a 1-click merge button for single-suggestion comments, while any suggestion from a multi-suggestion comment requires the PR author to manually apply whichever (if any) suggestion with which they agree. (My own preference here is the latter, `language tag`.)